internal/latestrelease/finder.go

```diff
@@ -74,30 +87,52 @@ func (s *latestReleaseVersionFinder) searchLatestVersionPerTool(currentVersion *
 	return false, nil, nil
 }
 
-func (s *latestReleaseVersionFinder) HasNewVersionAvailable(v, tool string) (newVersionAvailable bool, newVersion string, err error) {
-	if v == "" {
-		return false, "", nil
+func (f *latestReleaseVersionFinder) StoredLatestVersionAvailable() (needRefresh bool, foundVersion string, err error) {
+	latestVersionStored, _ := f.store.LoadLatestVersion()
+	v, err := semver.NewVersion(latestVersionStored)
```

Comment:
This line is a breaking change, please see my previous comment

Reply:
the latest stored version is already stored without the tag by using `versionFromTag`

Reply:
Yes, it is the wrong line